### PR TITLE
Configurable data store health check.

### PR DESCRIPTION
**Pull request by vjkoskela**

### CODE REVIEW

File: src/main/java/org/kairosdb/core/health/DatastoreQueryHealthCheck.java [hunk 1]
```diff
@@ -28,10 +58,12 @@ public String getName() {
 
     @Override
     protected Result check() throws Exception {
-        // TODO(ville): Make the query configurable
         try (final DatastoreQuery query = datastore.createQuery(
-                new QueryMetric(System.currentTimeMillis() - (10 * 60 * 1000),
-                        0, "jvm/threads/thread_count"))) {
+                new QueryMetric(
+                        System.currentTimeMillis() - (10 * 60 * 1000),
```
Comment (cwbriones) [inline src/main/java/org/kairosdb/core/health/DatastoreQueryHealthCheck.java:63]: Should we make this time (10 min) configurable as well?

File: src/test/java/org/kairosdb/core/health/DatastoreQueryHealthCheckTest.java [hunk 2]
```diff
@@ -26,15 +26,14 @@
     public void setup() throws DatastoreException {
```
Comment (cwbriones) [inline src/test/java/org/kairosdb/core/health/DatastoreQueryHealthCheckTest.java:26]: Can you add test(s) where
* Tags is nonempty and valid
* Tags is nonempty and invalid

File: src/main/java/org/kairosdb/core/health/DatastoreQueryHealthCheck.java (hunk 1 above)
Comment (cwbriones) [reply src/main/java/org/kairosdb/core/health/DatastoreQueryHealthCheck.java:63]: Actually, probably not necessary now that I think about it. I think the choice of metric matters more than lookback for a healthcheck.

Comment (vjkoskela) [reply src/main/java/org/kairosdb/core/health/DatastoreQueryHealthCheck.java:63]: That was my thinking as well, although it's easy to make it configurable.

**Review by cwbriones — APPROVED** [review]

**Pull request merged**